src/migrations/dapps/1718991797670_trades-and-asset-trades-table.ts

```diff
@@ -58,15 +61,59 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
         notNull: true
       },
       contract_address: { type: 'varchar(42)', notNull: true },
-      value: { type: 'numeric(78, 0)', notNull: true },
       beneficiary: { type: 'varchar(42)', notNull: false },
       extra: { type: 'text', notNull: false },
-      created_at: { type: 'timestamp', notNull: true, default: pgm.func('now()') }
+      created_at: { type: 'timestamptz(3)', notNull: true, default: pgm.func('now()::timestamptz(3)') }
+    }
+  )
+
+  pgm.createTable(
+    { schema: SCHEMA, name: TRADE_ASSETS_ERC721_TABLE },
+    {
+      asset_id: {
+        type: 'uuid',
+        notNull: true,
+        unique: true,
+        references: `"${TRADE_ASSETS_TABLE}"(id)`,
+        onDelete: 'CASCADE'
+      },
+      token_id: { type: 'text', notNull: true }
+    }
+  )
+
+  pgm.createTable(
+    { schema: SCHEMA, name: TRADE_ASSETS_ERC20_TABLE },
+    {
+      asset_id: {
+        type: 'uuid',
+        unique: true,
+        notNull: true,
+        references: `"${TRADE_ASSETS_TABLE}"(id)`,
+        onDelete: 'CASCADE'
+      },
+      amount: { type: 'numeric(78,0)', notNull: true }
```

Comment:
The amount bust be `greater than 0` and lower or equal than the maximum value an `uint256` can represent. There's a check [here](https://stackoverflow.com/questions/50072618/how-to-create-a-uint256-in-postgresql) that might be useful for the case.